src/db.ts

```diff
@@ -46,20 +47,20 @@ export class DB {
 
 	public createOrUpdateCollection<T extends TigrisCollectionType>(
 		collectionName: string, schema: TigrisSchema<T>): Promise<Collection<T>> {
-		return this.createOrUpdate(collectionName, schema,
+		return this.createOrUpdate(collectionName, CollectionType.DOCUMENTS, schema,
 			() => new Collection(collectionName, this._db, this.grpcClient));
 	}
 
 	public createOrUpdateTopic<T extends TigrisCollectionType>(
 		topicName: string, schema: TigrisSchema<T>): Promise<Topic<T>> {
-		return this.createOrUpdate(topicName, schema,
+		return this.createOrUpdate(topicName, CollectionType.MESSAGES, schema,
```

Comment:
it will be nice to cover this into RPC test. make the call and assert on server to have collection type set to messages.